content/workload-reduction-toolkit/how-the-toolkit-was-made.md

```diff
@@ -11,7 +11,7 @@ designed to support what you are already doing and can be adapted to work in you
 
 ## Background to the toolkit
 
-This toolkit is part of DfE’s action to support schools to reduce workload and address the causes of excessive workload
+This toolkit is part of DfE's action to support schools to reduce workload and address the causes of excessive workload
```

Comment:
hooold on... why is the only diff changing a good curly quote into a bad straight quote?